frontend/src/models/Pressure.ts

```diff
@@ -0,0 +1,14 @@
+export enum PressureStatus {
+    Normal = 'Normal',
+    Critical = 'Critical',
+    Error = 'Error',
+}
+
+export class Pressure {
+    status: PressureStatus = PressureStatus.Normal
+    value?: number
```

Comment:
```suggestion
    value: number
```
Why is this optional?